File: app/src/main/java/com/dominikgaller/alpinebooking/guide/core/inport/result/StartTourResult.java
package com.dominikgaller.alpinebooking.guide.core.inport.result;

/**
 * Output data carrier for the UC05 – StartTour use case.
 *
 * <p>SDD: See {@code documentation/use-cases/uc05-start-tour.spec.md}, section 3.
 */
public record StartTourResult(String status) {
}

File: src/main/kotlin/com/example/service/guide/core/inport/result/StartTourResult.kt
package com.example.service.guide.core.inport.result

/**
 * Output of the StartTour use case (UC05).
 *
 * SDD: see `documentation/ports/start-tour.inport.spec.md` § 2.2.
 */
data class StartTourResult(
    val status: String,
)
